### ISSUE TITLE
Jinjax fails with python3.9

### ISSUE THREAD

**Issue by mesemus** [jpsca/jinjax#59]
The recent version of jinjax fails with python3.9, even though pyproject.toml includes this version:

<img width="631" alt="image" src="https://github.com/jpsca/jinjax/assets/3385642/0414f798-3850-4117-adbd-9883e13e61b5">

**Comment by jpsca** [comment]
Yeah, my mistake. That is because  python3.9 doesn't support to replacing `typing.Union` with a pipe "|" by default... let's see if I can use an `import x from __future__" to fix it, because changing every type annotation to a string (option B) is tedious...

**Comment by mesemus** [comment]
No problem with that - we can freeze the version for the time being & migrate to 3.10+.
Just wanted to point this out in case others get the same issue.

And, as always, thanks for the library, the templates look much better & digestible for our
react ui folks.
